server/src/main/java/org/apache/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -152,9 +152,8 @@ private void assign(
       log.makeAlert("Tier[%s] has no servers! Check your cluster configuration!", tier).emit();
       return Collections.emptyList();
     }
-    predicate = predicate.and(s -> !s.isMaintenance());
-
-    return queue.stream().filter(predicate).collect(Collectors.toList());
+    Predicate<ServerHolder> general = s -> !s.isInMaintenance();
```

Comment:
Suggested "isGeneral" or "notInMaintenance"

Reply:
`isGeneral` is too abstract, `notInMaintenance` - having a negation in naming is usually not a good idea.

Reply:
isGeneral is the same as current general, just more conventional name for a Predicate object

Reply:
ah, I got it ✅